spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/Sample.scala

```diff
@@ -338,8 +338,6 @@ object Sample {
   def apply[T: ClassTag](
         featureTensor: Tensor[T],
         labelTensor: Tensor[T])(implicit ev: TensorNumeric[T]) : Sample[T] = {
-    require(featureTensor.isContiguous(), "featureTensor is not contiguous")
```

Comment:
if sparse tensor is contiguous, why remove this?

Reply:
it's duplicated checking, will check contiguous in ArraySample.apply